InvenTree/templates/js/translated/part.js

```diff
@@ -491,13 +491,16 @@ function duplicateBom(part_id, options={}) {
 }
 
 
+/*
+ * Construct a "badge" label showing stock information for this particular part
+ */
 function partStockLabel(part, options={}) {
 
     if (part.in_stock) {
         // There IS stock available for this part
 
         // Is stock "low" (below the 'minimum_stock' quantity)?
-        if (part.minimum_stock && part.minimum_stock > part.in_stock) {
+        if ((part.minimum_stock > 0) && (part.minimum_stock > part.in_stock)) {
```

Comment:
very curious that we are only discovering this now

Reply:
I do not think it was causing a bug, actually. But, there's a *potential* to cause a bug :)